docs/source-configuration.md

```diff
@@ -28,6 +28,20 @@ To run without auth file, use the following config:
 --source=kubernetes:http://kubernetes-ro?auth=""
 ```
 
+Alternatively, create a [service account](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/service_accounts.md)
+like this:
+
+```
+cat <EOF | kubectl create -f -
+apiVersion: v1beta3
+kind: ServiceAccount
+metadata:
+  name: heapster
+EOF
+```
+
+You will then need to reference the service account in your Heapster pod spec.
```

Comment:
Can you provide a pointer to an example PodSpec or may be some pod that uses service accounts?